content-src/asrouter/templates/SimpleSnippet/SimpleSnippet.schema.json

```diff
@@ -97,6 +97,20 @@
           "description": "Additional parameters for link action, example which specific menu the button should open"
         }
       }
+    },
+    "section_title_icon": {
```

Comment:
Just confirming, this is only spec'd for the simple snippet (no other templates?)

Reply:
Yep - Jean wants to tests this only on simple snippets for now. If this change has noticeable impact on interactions, she may want to roll it out to others, or maybe to all.